src/net_processing.cpp

```diff
@@ -681,6 +684,9 @@ class PeerManagerImpl final : public PeerManager
     /** Number of nodes with fSyncStarted. */
     int nSyncStarted GUARDED_BY(cs_main) = 0;
 
+    /** Hash of the last block we received via INV */
+    uint256 m_last_block_inv_triggering_headers_sync GUARDED_BY(cs_main) {};
```

Comment:
nit: Looks like, either:

* GUARDED_BY is not needed, as this is only accessed by a single thread (and if another thread is added in the future, it is unclear whether this should use cs_main or a different mutex)
* If it is needed, it might be best to use a mutex different from the validation mutex cs_main to avoid further bundling net_processing with chainstatemanager logic and fields.

Reply:
Ok, will drop the mutex.

Reply:
Removed.